app/src/main/res/layout/main_fragment.xml

```diff
@@ -174,10 +174,10 @@
                 android:layout_height="wrap_content"
                 android:completionThreshold="2"
                 android:imeOptions="actionSend|flagNoExtractUi"
-                android:inputType="text|textCapSentences"
+                android:inputType="text|textMultiLine"
+                android:maxLines="6"
                 android:paddingStart="12dp"
-                android:paddingEnd="8dp"
-                android:singleLine="true" />
+                android:paddingEnd="8dp" />
```

Comment:
I tinkered around with some paddings and I think having an additonal `android:paddingVertical="4dp"` improves some of the overlapping with the label.